sparrow.yaml

```diff
@@ -0,0 +1,29 @@
+app-id: com.craigrow.sparrow
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: sparrow
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+modules:
+  - name: sparrow
+    buildsystem: simple
+    build-commands:
+      - cp -R sparrow/bin/ /app/sparrow
+      - chmod +x /app/sparrow/sparrow
+      - mkdir /app/bin/
+      - mkdir /app/lib/
+      - ln -s /app/sparrow/sparrow /app/bin/sparrow
+      - install -Dm644 sparrow/com.craigrow.sparrow.metainfo.xml /app/share/appdata/com.craigrow.sparrow.metainfo.appdata.xml
+      - install -Dm644 sparrow/icons/256x256/com.craigrow.sparrow.png /app/share/icons/hicolor/256x256/apps/com.craigrow.sparrow.png
+      - install -Dm644 sparrow/com.craigrow.sparrow.desktop /app/share/applications/com.craigrow.sparrow.desktop
+    sources:
+      - type: archive
+        path: sparrow-wallet.tar.xz
+        sha256: 2ce299791449a53f46e5456b0dcd479d82a99093a9d6ebe197e4bef29592d819
+        dest: sparrow
```

Comment:
```suggestion
```

Reply:
How would you like me to change this field? This is the same name, I took it from the official example
![image](https://github.com/flathub/flathub/assets/80827297/839a31f1-b4de-4a68-aad3-b93c1249dc2b)

Reply:
You don't need it, you can just copy the bin and lib folder.

Reply:
a have a problem. i dont know how make that

Reply:
No, I decided everything, everything is fine